### ISSUE TITLE
Odyssey: KeyError 'Component' when outstanding mission during login

### ISSUE THREAD

**Issue by Athanasius** [EDCD/EDMarketConnector#964]
```
2021-04-01 21:40:10.983 - DEBUG - 16436:5832:5832 monitor.EDLogs.parse_entry:927: Invalid journal entry:
b'{ "timestamp":"2021-04-01T16:35:02Z", "event":"MissionCompleted", "Faction":"Adityan Regulatory State", "Name":"Mission_OnFoot_Collect_MB_name", "MissionID":737676998, "Commodity":"$PersonalDocuments_Name;", "Commodity_Localised":"Personal Documents", "Count":1, "Reward":23178, "MaterialsReward":[ { "Name":"CarbonFibrePlating", "Name_Localised":"Carbon Fibre Plating", "Category":"$MICRORESOURCE_CATEGORY_Component;", "Category_Localised":"Component", "Count":1 } ], "FactionEffects":[ { "Faction":"Adityan Regulatory State", "Effects":[  ], "Influence":[ { "SystemAddress":5031520572122, "Trend":"UpGood", "Influence":"+" } ], "ReputationTrend":"UpGood", "Reputation":"+" } ] }\r\n'
Traceback (most recent call last):
  File "C:\Users\Athan\Documents\Devel\EDMarketConnector\monitor.py", line 849, in parse_entry
    self.state[category][material] += reward.get('Count', 1)
KeyError: 'Component'
```

**Comment by Athanasius** [comment]
NB: This was for the prior Journal before I started the game up again.

**Comment by Athanasius** [comment]
Literal line from that Journal:

```json
{ "timestamp":"2021-04-01T16:35:02Z", "event":"MissionCompleted", "Faction":"Adityan Regulatory State", "Name":"Mission_OnFoot_Collect_MB_name", "MissionID":737676998, "Commodity":"$PersonalDocuments_Name;", "Commodity_Localised":"Personal Documents", "Count":1, "Reward":23178, "MaterialsReward":[ { "Name":"CarbonFibrePlating", "Name_Localised":"Carbon Fibre Plating", "Category":"$MICRORESOURCE_CATEGORY_Component;", "Category_Localised":"Component", "Count":1 } ], "FactionEffects":[ { "Faction":"Adityan Regulatory State", "Effects":[  ], "Influence":[ { "SystemAddress":5031520572122, "Trend":"UpGood", "Influence":"+" } ], "ReputationTrend":"UpGood", "Reputation":"+" } ] }```

**Comment by Athanasius** [comment]
I think it just needs 'Component' adding to the initialisation of self.state in monitor.py.  Will check for sure after some sleep.

**Comment by Athanasius** [comment]
Fixed in `develop` and 5.0.0-beta3 now
